h/app.py

```diff
@@ -85,15 +85,15 @@ def includeme(config):
     # retrieving services bound to the request.
     config.include("pyramid_services")
 
-    # Configure the transaction manager to support retrying retryable
-    # exceptions, and generate a new transaction manager for each request.
     config.add_settings(
         {
-            "tm.attempts": 3,
```

Comment:
tm.attempts is no longer needed because request retrying is now handled by the pyramid_retry extension, not pyramid_tm, and pyramid_retry does 3 attempts by default